command/e2etest/main_test.go

```diff
@@ -0,0 +1,296 @@
+package e2etest
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	tfcore "github.com/hashicorp/terraform/terraform"
+)
+
+var terraformBin string
+
+func TestMain(m *testing.M) {
+	if os.Getenv("GOOS") != "" || os.Getenv("GOARCH") != "" {
```

Comment:
Ahh yes, you're right. This was left over from some initial PoC code before I rearranged this to be `go test`-able. I'll remove it.